apps/ledger-live-mobile/src/components/RootNavigator/types/LendingEnableFlowNavigator.ts

```diff
@@ -71,6 +71,7 @@ export type LendingEnableFlowParamsList = {
     transaction?: Transaction;
     overrideAmountLabel?: string;
     hideTotal?: boolean;
+    isEdit?: boolean;
```

Comment:
I had to add this variable in every interface imported in EthereumFeeStrategy. We have the exact same interface in different navigators so yeah it's not good.